configs/run/quick_run.json

```diff
@@ -2,7 +2,8 @@
   "data" : ["/data/ztao/ttbarDiffXs13TeV/latest/mcWAlt/ttbar_nominal/mc16a/ttbar_0_pseudotop_parton_ejets.root"],
   "signal" : ["/data/ztao/ttbarDiffXs13TeV/latest/mcWAlt/ttbar_nominal/mc16a/ttbar_1_pseudotop_parton_ejets.root"],
   "observable_config" : "configs/observables/vars_ttbardiffXs_pseudotop.json",
-  "iterations" : 3,
+  "iterations" : 10,
+  "nruns":1,
```

Comment:
I guess this file is committed by accident instead of the default config for `--lrscheduler-config`: `configs/lrs/constant_warm_up.json`?